deps/libpcap/libpcap.MODULE.bazel

```diff
@@ -0,0 +1,18 @@
+# This file is generated. Do not hand edit.
+
+http_archive = use_repo_rule("//third_party/bazel/tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "libpcap",
+    files = {
+        "BUILD.bazel": "//deps/libpcap:overlay/overlay.BUILD.bazel",
+        "config.h": "//deps/libpcap:overlay/config.h",
+        "grammar.c": "//deps/libpcap:overlay/grammar.c",
+        "grammar.h": "//deps/libpcap:overlay/grammar.h",
+        "scanner.c": "//deps/libpcap:overlay/scanner.c",
+        "scanner.h": "//deps/libpcap:overlay/scanner.h",
```

Comment:
couldn't those be generated using `rules_flex` instead?

Reply:
The concern is nuanced, and mostly about if the lawyers consider importing the code via reference is different from vendoring it in. I use flex as one of the corner cases about how se up license rules. I can come back to this and add rules_flex once I get the license gathering from bazel right. Some thoughts on that here.
https://docs.google.com/document/d/1SHqUwT57t9rthj4GNHC-Pa_sST7zt99W_tK9Mo3RyrM/edit?tab=t.0#heading=h.ir6fquus2amu. I should add the split license case to that.